src/Core/MailTemplates/Handlebars/EmergencyAccessApproved.html.hbs

```diff
@@ -1,7 +1,7 @@
 ﻿{{#>FullHtmlLayout}}
-<table width="100%" cellpadding="0" cellspacing="0" style="margin: 0; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
```

Comment:
For these font removals, are we picking up the font from somewhere else? What if the email client chooses some nuts font for us?

Reply:
Font style is specified in the `<td>` tag where the text actually is.

Basically old layout 

```
<table font style>
  <tr font style>
    <td font style>
       text 
     </td>
   </tr>
</table>   
```

New layout 
```
<table>
  <tr>
    <td font style>
       text 
     </td>
   </tr>
</table>   
```